<launch>
    <!-- Sick LIDAR -->
    <node name="sick_tim571_2050101" pkg="sick_tim" type="sick_tim551_2050001"
            respawn="false" output="screen" launch-prefix="$(arg lidar_affinity_prefix)">
        <param name="min_ang" type="double" value="-1.81986"/>
        <param name="max_ang" type="double" value="1.81986"/>
        <param name="intensity" type="bool" value="true"/>
        <param name="skip" type="int" value="0"/>
        <param name="frame_id" type="str" value="sick_lidar"/>
        <param name="time_offset" type="double" value="-0.001"/>
        <param name="publish_datagram" type="bool" value="false"/>
        <param name="subscribe_datagram" type="bool" value="false"/>
        <param name="device_number" type="int" value="0"/>
        <param name="range_min" type="double" value="0.05"/>
        <param name="range_max" type="double" value="25.0"/>
        <param name="time_increment" type="double" value="0.000061722"/>
        <param name="hostname" type="string" value="192.168.20.99"/>
        <param name="port" type="string" value="2112"/>
        <param name="timelimit" type="int" value="5"/>
    </node>
</launch>